### PR TITLE
Fix: Asymetric icons are not displayed correctly inside the Nested Accordion - [ED-12168]

### PR DESCRIPTION
**Pull request by elchugreeva**

### CODE REVIEW

File: tests/playwright/sanity/modules/nested-accordion/nested-accordion-icon.test.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+			container = await editor.addElement( { elType: 'container' }, 'document' );
+
+		let nestedAccordionID,
+			nestedAccordion;
+
+		await test.step( 'Set horizontal icon & style size to 70', async () => {
+			// Act
+			await editor.closeNavigatorIfOpen();
+			nestedAccordionID = await editor.addWidget( 'nested-accordion', container );
+			nestedAccordion = await editor.selectElement( nestedAccordionID );
```
Comment (dmaziar) [inline tests/playwright/sanity/modules/nested-accordion/nested-accordion-icon.test.ts:44]:
```suggestion
			const nestedAccordion = await editor.selectElement( nestedAccordionID );
```

File: tests/playwright/sanity/modules/nested-accordion/nested-accordion-icon.test.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+			await editor.activatePanelTab( 'content' );
+			await page.locator( '.elementor-control-icons--inline__displayed-icon' ).first().click();
+			await page.locator( '#elementor-icons-manager__search > input' ).fill( 'address card' );
```
Comment (dmaziar) [inline tests/playwright/sanity/modules/nested-accordion/nested-accordion-icon.test.ts:47]:
```suggestion
			await page.locator( '#elementor-icons-manager__search  input' ).fill( 'address card' );
```
What if someone adds a div between #elementor-icons-manager__search  and input?

File: modules/nested-accordion/assets/scss/frontend.scss [hunk 3]
```diff
@@ -80,15 +80,17 @@
 
 					span {
 						height: var(--n-accordion-icon-size);
-						width: var(--n-accordion-icon-size);
-						font-size: var(--n-accordion-icon-size);
+						width: fit-content;
+					  	font-size: var(--n-accordion-icon-size);
```
Comment (hein-obox) [inline modules/nested-accordion/assets/scss/frontend.scss:84]: It looks like that you have added a few spaces.

File: modules/nested-accordion/assets/scss/frontend.scss [hunk 4]
```diff
@@ -80,15 +80,17 @@
...
 
 						& > i {
 							color: var(--n-accordion-icon-normal-color);
 						}
 
 						& > svg {
 							fill: var(--n-accordion-icon-normal-color);
+						  	width: var(--n-accordion-icon-size);
+						  	height: auto;
```
Comment (hein-obox) [inline modules/nested-accordion/assets/scss/frontend.scss:93]: Looking good.

**Review by dmaziar — DISMISSED** [review]

File: modules/nested-accordion/assets/scss/frontend.scss [hunk 5]
```diff
@@ -80,15 +80,17 @@
 
 					span {
```
[the remark this answers is not in the document]
Comment (hein-obox) [reply modules/nested-accordion/assets/scss/frontend.scss:81]: @elchugreeva Have you created a mindmap?

File: modules/nested-accordion/assets/scss/frontend.scss (hunk 3 above)
Comment (hein-obox) [reply modules/nested-accordion/assets/scss/frontend.scss:84]: Has this been updated?

File: modules/nested-accordion/assets/scss/frontend.scss (hunk 5 above)
[the remark this answers is not in the document]
Comment (hein-obox) [reply modules/nested-accordion/assets/scss/frontend.scss:81]: @Barcov Is the creation of mindmaps part of the onboarding process?

File: .run/watch-scripts.run.xml [hunk 6]
```diff
@@ -1,11 +0,0 @@
-<component name="ProjectRunConfigurationManager">
```
Comment (louiswol94) [inline .run/watch-scripts.run.xml:1]: @elchugreeva Was this file changed by accident or is it part of this diff on purpose?

File: modules/nested-accordion/assets/scss/frontend.scss [hunk 7]
```diff
@@ -89,6 +89,8 @@
 
 						& > svg {
 							fill: var(--n-accordion-icon-normal-color);
+						  	width: var(--n-accordion-icon-size);
+						  	height: auto;
```
Comment (hein-obox) [inline modules/nested-accordion/assets/scss/frontend.scss:93]: We are using a set `height` value in the span wrapper, and we are using `fit-content` for the `width`.

I am thinking could we follow that same pattern for the `svg` element?

It looks like here you have swopped the logic. Wouldn't it be easier to follow the pattern from the `span` parent?

File: modules/nested-accordion/assets/scss/frontend.scss [hunk 8]
```diff
@@ -89,6 +89,8 @@
 
 						& > svg {
 							fill: var(--n-accordion-icon-normal-color);
+						  	width: var(--n-accordion-icon-size);
```
Comment (hein-obox) [inline modules/nested-accordion/assets/scss/frontend.scss:92]: Maybe?

```suggestion
						  	width: inherit;
```

File: tests/playwright/sanity/modules/nested-accordion/helper.ts [hunk 9]
```diff
@@ -99,3 +99,17 @@ export async function setIconColor( editor, state, color, context ) {
...
+
+export async function addIcon( editor, page, iconName ) {
+	await editor.activatePanelTab( 'content' );
+	await page.locator( '.elementor-control-icons--inline__displayed-icon' ).first().click();
+	await page.locator( '#elementor-icons-manager__search input' ).fill( iconName );
+	await page.locator( '.elementor-icons-manager__tab__item' ).first().click();
+	await page.locator( '.dialog-insert_icon' ).click();
+}
+
+export async function setIconSize( editor, sizeInPx:string = '10' ) {
```
Comment (dmaziar) [inline tests/playwright/sanity/modules/nested-accordion/helper.ts:111]:
```suggestion
export async function setIconSize( editor: Editorage, sizeInPx:string = '10' ) {
```

File: tests/playwright/sanity/modules/nested-accordion/helper.ts [hunk 10]
```diff
@@ -99,3 +99,17 @@ export async function setIconColor( editor, state, color, context ) {
 		await editor.page.click( '.elementor-control-header_' + state + '_' + context );
 	}
 }
+
+export async function addIcon( editor, page, iconName ) {
```
Comment (dmaziar) [inline tests/playwright/sanity/modules/nested-accordion/helper.ts:103]: if you add types and there is typescript support in your IDE, so will all the errors and type mismatches. This is just a hint.

**Review by hein-obox — APPROVED** [review]

**Review by louiswol94 — APPROVED** [review]

**Pull request merged**